src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Email/SmtpConfigurationType.php

```diff
@@ -47,24 +48,30 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 'required' => false,
                 'empty_data' => '',
                 'label' => $this->trans('Email domain name', 'Admin.Advparameters.Feature'),
+                'multistore_configuration_key' => 'PS_MAIL_DOMAIN',
                 'help' => $this->trans('Fully qualified domain name (keep this field empty if you don\'t know).', 'Admin.Advparameters.Help'),
             ])
             ->add('server', TextType::class, [
                 'required' => false,
                 'label' => $this->trans('SMTP server', 'Admin.Advparameters.Feature'),
+                'multistore_configuration_key' => 'PS_MAIL_SERVER',
                 'help' => $this->trans('IP address or server name (e.g. smtp.mydomain.com).', 'Admin.Advparameters.Help'),
             ])
             ->add('username', TextType::class, [
                 'required' => false,
                 'empty_data' => '',
                 'label' => $this->trans('SMTP username', 'Admin.Advparameters.Feature'),
+                'multistore_configuration_key' => 'PS_MAIL_USER',
                 'help' => $this->trans('Leave blank if not applicable.', 'Admin.Advparameters.Help'),
+                'attr' => ['autocomplete' => 'new-password'],
```

Comment:
Not sure about this one? 🤔
```suggestion
```

Reply:
Hello, 

The field is automatically filled with the shop password, which is very confusing for the merchant